lib/matplotlib/axes/_axes.py

```diff
@@ -6952,6 +6987,11 @@ def hist2d(self, x, y, bins=10, range=None, density=False, weights=None,
         alpha : ``0 <= scalar <= 1`` or ``None``, optional
             The alpha blending value.
 
+        **kwargs
+            Additional parameters not listed here are passed along to the
```

Comment:
```suggestion
            Additional parameters are passed along to the
```
Does not add information.